languages/groovy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Install groovy, http://www.groovy-lang.org/
+#
+# Add at least the following environment variables to your project configuration
+# (otherwise the defaults below will be used).
+# * GROOVY_VERSION
+#
+# Include in your builds via
+# source /dev/stdin <<< "$(curl -sSL https://raw.githubusercontent.com/codeship/scripts/master/languages/groovy.sh)"
+GROOVY_VERSION=${GROOVY_VERSION:="2.4.11"}
+GROOVY_PATH=${GROOVY_PATH:=$HOME/groovy}
+CACHED_DOWNLOAD="${HOME}/cache/groovy-v${GROOVY_VERSION}.zip"
+
+mkdir -p "${GROOVY_PATH}"
+
+wget --continue --output-document "${CACHED_DOWNLOAD}" "https://bintray.com/artifact/download/groovy/maven/apache-groovy-binary-${GROOVY_VERSION}.zip"
+unzip -q -o "${CACHED_DOWNLOAD}" -d "${GROOVY_PATH}"
```

Comment:
It would be great if we can unzip Groovy to GROOVY_PATH and not a subdirectory. tar supports a --strip-components option, which is very useful for this, but I'm not sure if zip offers something similar.